### DISCUSSION TITLE
Move completed job from archive back to job table and re-run it

### DISCUSSION THREAD

**Discussion by pulpdood** [timgit/pg-boss#392] [Q&A]
This may sound like an anti-pattern or odd request, but the situation we have is that a part of our code had created jobs which were run and marked as completed, but without performing the task we want because the job handler code logic was incorrect. 

We have since updated the job handler code, and we want to re-run the previously mentioned jobs on the new job handler code, but unfortunately the jobs are now in the archive table, and we're not sure how to re-trigger these jobs to run.

**Comment by timgit** [comment]
It sounds like you should run an INSERT..SELECT command
